src/components/Button/Button.scss

```diff
@@ -167,7 +167,6 @@
           @include theme-prop(background-color, disabled-background-color);
           @include theme-prop(color, disabled-text-color);
           cursor: not-allowed;
-          pointer-events: none;
```

Comment:
fix bug on button: disabled cursor not displayed when button is disable

Reply:
I check it and the functionality works as planned because of the button still have disabled attribute :)